findFamily.php

```diff
@@ -53,13 +80,51 @@
         <h1>Search Family Account</h1>
```

Comment:
I would add somewhere some direction for the user. Say something like 'Please fill out at least one search field'